### ISSUE TITLE
Uncollapsing profile list is not possible using keyboard

### ISSUE THREAD

**Issue by makisukurisu** [UnigramDev/Unigram#2813]
**Describe the bug**
It's not possible to reveal added accounts using just keyboard. And this leads to next problem - it's not possible to add another account without keyboard (assuming you have collapsed account list, which you have by default)

I'm talking about account list in "Navigation Menu"

**To Reproduce**
1. Collapse account list/make a fresh install
2. Enable some sort of accessibility app (Like Narrator)
3. Open Navigation Menu
4. Fail to find a way to focus on collapsed account
5. Fail to add another account/change account

**Expected behavior**
To be able to focus on this element and to uncollapse it

**Screenshots**
Default focus after opening Navigation Menu:
![image](https://user-images.githubusercontent.com/49622129/187061757-cce845c1-b8d7-47cf-9c69-b13396ae8c9f.png)

**Version Info**
 - Unigram: v. 8.9 7575 x64 Direct (I guess store one has the same problem)
 - Windows: 22H2 (22000.856)

**Additional context**
Initially reported by @gamov1995

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/bf7403fb9c4da49ae15111b0adacba2e25f48e65
